test/services/audit/AuditServiceSpec.scala

```diff
@@ -18,6 +18,7 @@ package services.audit
 
 import base.{Injector, TestHelper}
 import config.AppConfig
+import models.ExportsDeclaration.Audit.format
```

Comment:
You sure the connector doesn't need this import too?

Reply:
Yes. Without the explicit imports, we would have compilation failures